src/components/AppComponents/AppHeader/index.tsx

```diff
@@ -24,14 +24,15 @@ export default function AppHeader(props) {
             </a>
           </div>
           <div className="govuk-header__content">
-            <a href="#" className="hmrc-header__service-name hmrc-header__service-name--linked" id="service-header">
+            <a href="/" className="hmrc-header__service-name hmrc-header__service-name--linked" id="service-header">
```

Comment:
undo this change, this will create issue in higher env, where we have a path for our application

Reply:
Thanks, resolved here [e312177](https://github.com/norm-l/odx/pull/123/commits/e312177a419463cdc4dbb3759f01a8bebda331e7)